neo/public/stylesheets/main.css

```diff
@@ -128,11 +123,37 @@ search-bar {
     color : white;
 }
 
-.number{
-  color: #2795ee;
+.stats{
   position: absolute;
-  bottom:6%;
-  left:1%;
-  width:10%;
-  z-index:0;
-}
\ No newline at end of file
+  bottom:0;
+  pointer-events: none;
+  line-height: 1;
+  width: 100%;
+}
+
+time-series {
+  z-index: 1000;
+  pointer-events: auto;
+  position: relative;
+  display: inline-block;
+  left: 4%;
+}
+
+#count-div{
+  color: #000000;
+  font-weight:500;
+  margin-top: 20px;
+  margin-bottom: 40px;
+  padding-left: 10px;
+  float: left;
+}
+
+#count{
+  font-size: 32px;
+  margin-bottom: 0;
+  text-align: right;
```

Comment:
@chenlica I found @HotLemonJuice specify the alignment in this CSS code without using a mini table. I think it's good enough. What's your opinion on this?